manifests/init.pp

```diff
@@ -813,6 +825,14 @@
     validate_re($sshd_config_permittunnel_real, '^(yes|no|point-to-point|ethernet|unset)$', "ssh::sshd_config_permittunnel may be either 'yes', 'point-to-point', 'ethernet', 'no' or 'unset' and is set to <${sshd_config_permittunnel_real}>.")
   }
 
+  if $sshd_config_hostcertificate_real != undef {
+    validate_absolute_path($sshd_config_hostcertificate_real)
+  }
+
+  if $sshd_config_trustedusercakeys_real != undef {
+    validate_absolute_path($sshd_config_trustedusercakeys_real)
```

Comment:
https://www.freebsd.org/cgi/man.cgi?sshd_config(5)

This can also be `'none'`.